src/wcm/__main__.py

```diff
@@ -166,4 +166,17 @@ def publish(component, profile="default", debug=False, dry_run=False):
 def download(id, profile="default",file_path = None):
     logging.info("Downloading component")
     _download.download(id, profile=profile,download_path=file_path)
-    print("done")
\ No newline at end of file
+    print("done")
```

Comment:
Use ` click.secho(f"Success", fg="green")` instead of `print`. Generally not a good practice to use `print`.

Reply:
Just wait until you see how I took user input. Just out of curiosity why is print bad practice?

Reply:
1. Unlike loggers, prints offer little control, like they cannot be filtered on levels like ERROR etc. 

2. Usually OK for CLIs (on outermost functions), but if code is imported in a server like tomcat, print statement go to server specific logs, and get separated from application log.

3. CLIs output content to stdout only when it is the command's final output, and messages like done success go in logs. Example: For the list command, the output should just be the component tree, success/done should go to log. With that output could be redirected to file and post processed, say list had an option --output-format json|yaml, then someone could consume it, but if it had Done, Success, etc parsing would fail.

Reply:
Of course, good to follow projects norms, `click.echo` in some and `print` in other places would make refactoring code time consuming.